test/Graphics/Transform.cpp

```diff
@@ -6,17 +6,22 @@
 #include <GraphicsUtil.hpp>
 #include <vector>
 
-namespace std
+// Use StringMaker to avoid opening namespace std
+namespace doctest
 {
-std::ostream& operator<<(std::ostream& out, const std::vector<float>& vector)
+template <>
+struct StringMaker<std::vector<float>>
 {
-    out << "{ ";
-    for (size_t i = 0; i + 1 < vector.size(); ++i)
-        out << vector[i] << ", ";
-    out << vector.back() << " }";
-    return out;
-}
-} // namespace std
+    static String convert(const std::vector<float> vector)
```

Comment:
You probably meant `const std::vector<float>& vector` here (const-reference).

Reply:
Thanks!